fem/fe_coll.cpp

```diff
@@ -260,10 +260,30 @@ FiniteElementCollection *FiniteElementCollection::New(const char *name)
    }
    else if (!strncmp(name, "NURBS", 5))
    {
-      fec = new NURBSFECollection(atoi(name + 5));
+      int Order = atoi(name + 5);
+
+      if (Order < 10)
+      {
+         fec = new NURBSFECollection(Order);
+      }
+      else if (Order > 100)
+      {
+         int px = (Order - Order%100)/100;
+         Order -= px*100;
+         int py = (Order - Order%10)/10;
+         Order -= py*10;
+         int pz = Order;
+         fec = new NURBSFECollection(px,py,pz);
+      }
+      else
+      {
+         mfem_error("FiniteElementCollection::New : "
+                    "Unknown NURBS ORDER!");
+      }
    }
    else
    {
+      cout<<"NAME = "<<name<<endl;
```

Comment:
Remove before merging

Reply:
Will do that.

Reply:
I have pulled the changes and see you already made the changes...